### PR TITLE
increase gap in containerStyle

### PR DESCRIPTION
**Pull request by aashikofsach**
change value of gap

### CODE REVIEW

File: src/StarRating.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 const containerStyle = {
   display: "flex",
   alignItems: "center",
-  gap: "16px",
+  gap: "20px",
```
Comment (adityasingh234) [inline src/StarRating.js]: Gap should not be more than 18px. Keep it between 16-18.
`gap:17px;`

File: src/StarRating.js [hunk 2]
```diff
@@ -13,13 +13,13 @@ const starContainerStyle = {
...
-  color : PropTypes.string,
-  size : PropTypes.number,
-  messeges : PropTypes.array,
-  className : PropTypes.string,
-  onSetRating : PropTypes.func,
+  maxRating: PropTypes.number,
+  defaultRating: PropTypes.number,
+  color: PropTypes.string,
+  size: PropTypes.number,
+  messeges: PropTypes.array,
```
Comment (adityasingh234) [inline src/StarRating.js]: Correct the spelling of messages. It should be `messages`.

File: src/StarRating.js [hunk 3]
```diff
@@ -29,19 +29,19 @@ export default function StarRating({
...
 }) {
   const [rating, setRating] = useState(defaultRating);
   const [tempRating, setTempRating] = useState(defaultRating);
 
   const textStyle = {
     lineHeight: "1",
     margin: "0",
-    color:color,  // when that kind of case become of assigning value we can directly write 'color' only 👍
-    fontSize :`${size/1.5}px`,
+    color: color,  // when that kind of case become of assigning value we can directly write 'color' only 👍
```
Comment (adityasingh234) [inline src/StarRating.js:43]: Delete this comment from here. Move it to the top of the object.

**Review by adityasingh234 — CHANGES_REQUESTED** [review]

File: src/StarRating.js (hunk 3 above)
Comment (aashikofsach) [reply src/StarRating.js:43]: Probably we not need to do that.

Comment (adityasingh234) [reply src/StarRating.js:43]: Okay no problem

**Review by adityasingh234 — APPROVED** [review]

**Pull request merged**